src/kernel.ts

```diff
@@ -633,10 +633,11 @@ class Kernel implements IKernel {
       session: this._clientId
     }
     var defaults = {
-      silent : true,
-      store_history : false,
+      silent : false,
+      store_history : true,
       user_expressions : {},
-      allow_stdin : false
+      allow_stdin : false,
```

Comment:
This is listed as `true` here: http://jupyter-client.readthedocs.org/en/latest/messaging.html#execute